pkg/packet/pcep/object.go

```diff
@@ -22,6 +21,24 @@ const (
 	RFC_COMPLIANT
 )
 
+// Determine PCC type from capability
+func DetectPccType(caps []CapabilityInterface) (pccType PccType) {
```

Comment:
Shall the function name be "DeterminePccType" to match the process description and comments?